docs/examples/promise/03_listen_to_balance_change/index.js

```diff
@@ -8,27 +10,31 @@ async function main () {
   // Create an await for the API
   const api = await ApiPromise.create();
 
-  // Retrieve the initial balance. Since the call has no callback, it is simply a promise
-  // that resolves to the current on-chain value
-  let previous = await api.query.balances.freeBalance(Alice);
+  // Retrieve the current on-chain balance. The call has a callback since WebSockets 
+  // subscriptions require a handler.
+  let previous = await api.query.balances.freeBalance(Alice, (previous) => {
+    return new BN(previous);
```

Comment:
Nope. Balance already extends `BN`. So this is not very useful and just confuses. We don't need to import `BN` and/or used it at all.